tests/osgi-tests/src/test/java/org/wso2/is/portal/user/client/api/unit/test/IdentityStoreClientServiceTest.java

```diff
@@ -394,7 +394,7 @@ public void testListUsersByOffsetAndLength() throws UserPortalUIException {
         List<UserListBean> users = identityStoreClientService.listUsers(1, 2, PRIMARY_DOMAIN, requestedClaims);
 
         Assert.assertNotNull(users, "Failed to list the users.");
-        Assert.assertTrue(!users.isEmpty() && users.size() == 2, "Number of users received in the response " +
+        Assert.assertTrue(!users.isEmpty() && users.size() == 1, "Number of users received in the response " +
```

Comment:
Here, 
1. !users.isEmpty() is redundant as users.size() == 1
2. use Assert.assertEquals(users.size() , 1, "Number of users received in the response should match"). See the error printed if you change "1" to "2".